src/components/AlchemyRow.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <td class="columnSpan">
+    <GameAsset class="centered"
+      :image="Assets.FromDir(`${props.group}${props.idx+1}`, 'alchemy')"
```

Comment:
You don't need to namespace props with `props.propName`. You can just use `propName` in templates.

```suggestion
      :image="Assets.FromDir(`${group}${idx+1}`, 'alchemy')"
```